server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/data/CassandraDomainListModule.java

```diff
@@ -57,9 +63,18 @@ public CassandraDomainListConfigurationPerformer(ConfigurationProvider configura
         }
 
         @Override
-        public void initModule() throws Exception {
-            cassandraDomainList.setLog(LOGGER);
-            cassandraDomainList.configure(configurationProvider.getConfiguration("domainlist"));
+        public void initModule() {
+            try {
+                cassandraDomainList.setLog(LOGGER);
+                cassandraDomainList.configure(configurationProvider.getConfiguration("domainlist"));
+            } catch (ConfigurationException e) {
+                Throwables.propagate(e);
+            }
```

Comment:
All those methods are called only when starting.
So throwing Runtime at lower level ensure that we break earlier.